app/views/smart_answers/shared/_hide_this_page.html.erb

```diff
@@ -0,0 +1,23 @@
+<% content_for :hide_this_page_banner do %>
+  <% if presenter.use_hide_this_page? %>
+    <%= render "components/hide-this-page-banner", {
+      link_url: destroy_flow_path(ext_r: "true"),
+      link_label: "Hide this page",
```

Comment:
Minor: you don't need this line anymore.

Reply:
Now removed